Source/Core/VideoBackends/D3D/D3DState.h

```diff
@@ -48,6 +48,15 @@ union SamplerState {
   u64 packed;
 };
 
+union DepthState {
```

Comment:
I followed the style that was already here and lint didn't complain. But you are right, the contribution guide says the following:

> The opening brace for namespaces, classes, functions, enums, structs, unions, conditionals, and loops go on the next line.